galaxy_ng/app/settings.py

```diff
@@ -30,6 +30,10 @@
 # FIXME(cutwater): 1. Rename GALAXY_API_ROOT in pulp-ansible to ANSIBLE_API_ROOT
 #                  2. Rename API_PATH_PREFIX to GALAXY_API_ROOT
 GALAXY_API_PATH_PREFIX = "/api/galaxy"
+
+# Location where the pulp api will be served
+API_ROOT = "/api/galaxy/pulp/"
```

Comment:
I'm a little concerned about pulling the rug out from under the existing `/pulp/` endpoints. We have no idea if on prem customers are using these endpoints or not and it seems like breaking the existing behavior of the API could be a bad idea. It might be a good idea to return a `301 moved permanently` on the old `/pulp/api/v3/**` endpoints or something like that.

Reply:
Agreed we should add a redirect